pkg/trace/transform/transform.go

```diff
@@ -178,6 +179,20 @@ func OtelSpanToDDSpan(
 	return ddspan
 }
 
+// escapeJSONString writes a string s to the stringbuilder sb, ensuring quotation marks are escaped
+func escapeJSONString(s string, sb *strings.Builder) {
```

Comment:
Can we import the `""encoding/json"` package so that it can handle all the various escape character sequences we'll need? I worry about messing it up ourselves when we run into other characters like `/`, `\`, `\n`, etc

Reply:
I imagine it would look something like this:
```
func jsonString(v interface{}) string {
	bytes, err := json.Marshal(v)
	if err != nil {
		panic(err) // Handle the error as appropriate for your use case
	}
	return string(bytes)
}
```